docs/external_contribution_request.md

```diff
@@ -0,0 +1,17 @@
+## Issue title
+
+Publish `EXTENSION_NAME` to Open VSX

```

Comment:
Should we just use this as the title, otherwise the copy/paster has to update/remove the placeholder title? 

```suggestion
## Publish `EXTENSION_NAME` to Open VSX

```

Reply:
Hmm I think it wouldn't exactly make sense this way, because there is also a heading saying `Issue Body`. I think it will be even better if we make the issue title a code block the same way as the issue body.

We could also simply have the whole md file be the issue content (make the first heading imply to be used as the issue title), but in this way we can both add notes to the sections (maybe some guidance to the contributors), as well as providing a simple copying experience thanks to the code blocks.

Reply:
I implemented it as I said here, but we can change it anytime if it does not make sense to you 🙂.